src/plugins/chart_expressions/expression_gauge/public/components/gauge_component.tsx

```diff
@@ -100,18 +111,18 @@ function normalizeBands(
 }
 
 function getTitle(
-  labelMajorMode: GaugeLabelMajorMode,
-  labelMajor?: string,
+  majorMode?: GaugeLabelMajorMode | GaugeCentralMajorMode,
```

Comment:
nit: I am not really convinced about this name change. Wouldn't it be better to just leave it as `labelMajorMode`? In the end it's still "major", most important label in the chart.